python/tests/test_search.py

```diff
@@ -0,0 +1,46 @@
+from sidecar import search
+
+def test_search(monkeypatch):
+    def mock_search_paper(*args, **kwargs):
+        response = [
+            {
+                "title": "Sample Paper Title",
+                "abstract": "Sample Abstract",
+                "venue": "Sample Venue",
+                "year": 2021,
+                "paperId": "sample-id-1",
+                "citationCount": 10,
+                "openAccessPdf": "https://sample.org",
+                "authors": [
+                    {"authorId": "auth-id-1", "name": "Sample Author Name 1"},
+                    {"authorId": "auth-id-2", "name": "Sample Author Name 2"},
+                ]
+            },
+            {
+                "title": "Sample Paper Title 2",
+                "abstract": "Sample Abstract 2",
+                "venue": "Sample Venue 2",
+                "year": 2022,
+                "paperId": "sample-id-2",
+                "citationCount": 20,
+                "openAccessPdf": "https://sample.org/2",
+                "authors": [
+                    {"authorId": "auth-id-3", "name": "Sample Author Name 3"},
+                    {"authorId": "auth-id-4", "name": "Sample Author Name 4"},
+                ]
+            },
+        ]
+        return response
+
+    monkeypatch.setattr(search.Search, "search_paper", mock_search_paper)
+
+    searcher = search.Searcher()
+    result = searcher.s2.search_paper("any-query-string-you-like")
```

Comment:
Change this to test `search.search`

Reply:
resolved in https://github.com/refstudio/refstudio/pull/250/commits/b716691359753cba88ef5d0ef28f3b73e494fe8c